ui.frontend/src/utils.js

```diff
@@ -189,4 +190,22 @@ export default class Utils {
             }
         }
     }
+
+    /**
+     * for backward compatibility with older data formats of prefill services like FDM
+     * @param prefillJson
+     * @returns {{data}|*}
+     */
+    static stripIfWrapped(prefillJson) {
+        if (prefillJson && prefillJson.hasOwnProperty("data")) {
+            const data = prefillJson.data;
+            if (data && data.hasOwnProperty("afData")) {
+                const afData = data.afData;
+                if (afData && afData.hasOwnProperty("afBoundData")) {
+                    return afData.afBoundData;
+                }
+            }
+        }
+        return prefillJson;
```

Comment:
I think we should support this ootb as well. Unbound fields will not get prefilled here.